src/ModelXSD/StartupServicesModel.xsd

```diff
@@ -649,4 +654,44 @@
     </xs:choice>
   </xs:group>
 
+  <xs:attributeGroup name="StartupNameValuePair">
+    <xs:annotation>
+      <xs:documentation>Name and Value defined as an attribute.</xs:documentation>
+    </xs:annotation>
+    <xs:attribute name="Name" use="required">
+      <xs:annotation>
+        <xs:documentation>The name of the setting to override.</xs:documentation>
+      </xs:annotation>
+      <xs:simpleType>
+        <xs:restriction base="xs:string">
+          <xs:minLength value="1"/>
+        </xs:restriction>
+      </xs:simpleType>
+    </xs:attribute>
+    <xs:attribute name="Value" type="xs:string" use="required">
+      <xs:annotation>
+        <xs:documentation>The new value of the setting.</xs:documentation>
+      </xs:annotation>
+    </xs:attribute>
+  </xs:attributeGroup>
+
+  <!-- Type for the Service Instance Definition file generated by Visual studio tools for application deployment. -->
+  <xs:complexType name="StartupServiceInstanceDefinitionType">
+    <xs:sequence>
+      <xs:element name="Parameters">
+        <xs:annotation>
+          <xs:documentation>List of service parameters for the application as defined in startup service manifest and their respective values.</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+          <xs:sequence>
+            <xs:element name="Parameter" minOccurs="0" maxOccurs="unbounded">
+              <xs:complexType>
+                <xs:attributeGroup ref="StartupNameValuePair"/>
```

Comment:
what is startup name ?

Reply:
Parameters inside ServiceConfig file are of type Name and Value pair. This type is been defined as StartupNameValuePair  to separate it out from the NameValuePair defined in ServiceFabricServiceModel.xsd

Reply:
should it be StartupServiceNameValuePair

Reply:
ServicesManifest  has been termed as StartupServicesManifest, similarly Startup has been added prefix to distinguish in other types as well in XSD e.g. StartupStatelessServiceType